src/OrbitGl/TimerTrack.cpp

```diff
@@ -83,17 +83,15 @@ WorldXInfo ToWorldX(double start_us, double end_us, double inv_time_window, floa
 }  // namespace
 
 void TimerTrack::DrawTimer(TextBox* prev_text_box, TextBox* current_text_box,
-                           TextBox* next_text_box, uint64_t min_tick, uint64_t max_tick,
-                           float z_offset, Batcher* batcher, GlCanvas* canvas, float world_start_x,
-                           float world_width, double inv_time_window, bool is_collapsed, float z,
-                           const TextBox* selected_textbox, uint64_t highlighted_function_id,
-                           uint64_t pixel_delta_in_ticks, uint64_t min_timegraph_tick,
+                           TextBox* next_text_box, const internal::DrawData& draw_data,
```

Comment:
Nit: Put function arguments that are input only first.

https://google.github.io/styleguide/cppguide.html#Inputs_and_Outputs

Reply:
Which arguments do you mean? the text_boxes are input only. The only output arguments are the last two, or am I missing something?

Reply:
If I see it right, there are calls to SetPos(...) and SetSize(...) at least on current_text_box (which I guess makes it an input/output parameter). But it's hard to read what is input / output here due to many pointers being used (the standard Google code style would be: Input type should be const& or by value, input/output or just output would be *).

Reply:
Thanks, missed that part for the "current". I moved it to the other output parameters and made the other text boxes "const".